src/io/matpower.jl

```diff
@@ -546,7 +546,7 @@ end
 
 "adds dcline costs, if gen costs exist"
 function add_dcline_costs(data::Dict{String,Any})
-    if length(data["gencost"]) > 0 && length(data["dclinecost"]) <= 0
+    if length(data["gencost"]) > 0 && length(data["dclinecost"]) <= 0 && length(data["dcline"]) > 0
```

Comment:
This fix belongs in PowerModels master.

Reply:
I did a cherrypick to update these lines to the current version in master and implement the fix for PWL costs in DC-lines. How would you prefer to update these files in the `moi-julia-v0.7`?